Bump sane-pdf-reports docker image (#22256)
demisto/content: Packs/Base/ReleaseNotes/1_31_33.md

#### Scripts
##### SanePdfReports

> - Updated the Docker image to: *demisto/sane-pdf-reports:1.0.0.37454*.
>     - Fixed an issue where images didn't display well.
>     - Fixed an issue where applying color to markdown made the instance page stuck.

Reviewer: - Fixed an issue where images didn't display well - in what case/scenario?
```suggestion
- Updated the Docker image to: *demisto/sane-pdf-reports:1.0.0.37454*.
- Fixed an issue where images did not display well.
- Fixed an issue where applying color to a markdown based report affected the instance.
```
